android/app/src/main/AndroidManifest.xml

```diff
@@ -31,5 +31,13 @@
         <meta-data
             android:name="flutterEmbedding"
             android:value="2" />
+
+        <receiver
+            android:name=".FontReceiver"
+            android:exported="true">
+            <intent-filter>
+                <action android:name="com.tavultesoft.kmapro.keyboard_changed"/>
```

Comment:
Do we also want to include another action for `com.tavultesoft.kmapro.debug.keyboard_changed`?

So the browser would detect either debug or release versions of Keyman